src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -501,22 +501,18 @@ public final class JavadocTokenTypes {
      * <pre>
      * <code> |--JAVADOC_INLINE_TAG[1x0] :
      *               [{&#64;link org.apache.utils.Lists.Comparator#compare(Object)}]
-     *        |--JAVADOC_INLINE_TAG_START[1x0] : [{]
-     *        |--LINK_LITERAL[1x1] : [@link]
-     *        |--WS[1x6] : [ ]
-     *        |--REFERENCE[1x7] : [org.apache.utils.Lists.Comparator#compare(Object)]
-     *            |--PACKAGE_CLASS[1x7] : [org.apache.utils]
-     *            |--DOT[1x23] : [.]
-     *            |--CLASS[1x24] : [Lists]
-     *            |--DOT[1x29] : [.]
-     *            |--CLASS[1x30] : [Comparator]
-     *            |--HASH[1x40] : [#]
-     *            |--MEMBER[1x41] : [compare]
-     *            |--PARAMETERS[1x48] : [(Object)]
-     *                |--LEFT_BRACE[1x48] : [(]
-     *                |--ARGUMENT[1x49] : [Object]
-     *                |--RIGHT_BRACE[1x55] : [)]
-     *        |--JAVADOC_INLINE_TAG_END[1x56] : [}]
+     *        |--JAVADOC_INLINE_TAG_START -> {
```

Comment:
Please do escape of `>`

Reply:
Doing it right away thanks!

Reply:
I have done the necessary changes @romani kindly review it, Thanks!